fc/mef.py

```diff
@@ -710,9 +479,9 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
     3. Some subpopulations are then discarded if they are close to either
        the minimum or the maximum channel value. In addition, if the MEF
        value of some subpopulation is unknown (represented as a ``NaN`` in
-       `peaks_mef`), the whole subpopulation is also discarded.
+       `mef_values`), the whole subpopulation is also discarded.
     4. The measured fluorescence of each subpopulation is compared with
-       the known MEF values in `peaks_mef`, and a standard curve function
+       the known MEF values in `mef_values`, and a standard curve function
        is generated using the appropriate MEF model.
```

Comment:
In step 2 of Notes, "of each subpopulation" and "for each cluster" is confusingly redundant.